docs/config.c

```diff
@@ -1099,14 +1099,14 @@
 ** .pp
 ** This option can be enabled on the command line, "neomutt -d 2"
 ** .pp
-** See also: \fC$$debug_file\fP
+** See also: \fI$$debug_file\fP
 */
 
 { "default_hook", DT_STRING, "~f %s !~P | (~P ~C %s)" },
 /*
 ** .pp
 ** This variable controls how some hooks are interpreted if their pattern is a
-** plain string or a regex. i.e. they don't contain a pattern, like \fC~f\fP
+** plain string or a regex. i.e. they don't contain a pattern, like \fI~f\fP
```

Comment:
This is a literal, so I'd use bold, I think.  It's not a clear case, I think, though.